src/components/inngang-paabegynt/InngangPaabegynt.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button } from "@navikt/ds-react";
+import { useRouter } from "next/router";
+import { useState } from "react";
+import api from "../../api.utils";
+import { deleteSoknad } from "../../api/deleteSoknad-api";
+import { IPaabegyntSoknad } from "../../types/quiz.types";
+
+export function InngangPaabegynt(paabegynt: IPaabegyntSoknad) {
+  const router = useRouter();
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [isNavigatingToSoknad, setIsNavigatingToSoknad] = useState(false);
+  const [hasCreateNewSoknadError, SetHasCreateNewSoknadError] = useState(false);
+  const [hasDeleteSoknadError, SetHasDeleteSoknadError] = useState(false);
+
+  function navigateToSoknad() {
+    setIsNavigatingToSoknad(true);
+    router.push(`/${paabegynt.uuid}`);
+  }
+
+  async function deleteAndCreateSoknad() {
+    setIsLoading(true);
+    const deleteSoknadResponse = await deleteSoknad(paabegynt.uuid);
+
+    if (deleteSoknadResponse.ok) {
+      createNewSoknad();
+    } else {
+      setIsLoading(false);
+      SetHasDeleteSoknadError(true);
+      throw new Error(deleteSoknadResponse.statusText);
+    }
+  }
+
+  async function createNewSoknad() {
+    try {
+      setIsLoading(true);
+      const uuidResponse = await fetch(api("soknad/get-uuid"));
+
+      if (uuidResponse.ok) {
+        const uuid = await uuidResponse.text();
+        router.push(`/${uuid}`);
+      } else {
+        setIsLoading(false);
+        throw new Error(uuidResponse.statusText);
+      }
+    } catch (error) {
+      // TODO Sentry log
+      // eslint-disable-next-line no-console
+      console.error(error);
+      setIsLoading(false);
+      SetHasCreateNewSoknadError(true);
+    }
+  }
+
+  return (
+    <>
+      Påbegynt soknad : {paabegynt.startDato}
+      <br />
+      <Button variant="primary" onClick={navigateToSoknad} loading={isNavigatingToSoknad}>
```

Comment:
Denne kan kanksje være en lenke, slik som dette: ```<Button as="a" href={`/${paabegynt.uuid}`}>Forsett</Button>```. Siden den ikke gjør noe annet enn å navigere. Men det er småpirk, og sannsynligvis noe som kommer til å endre seg når designet kommer på plass. :)

Reply:
Ja ikke sant. Jeg ser at du fremdeles har onClick on loading på knappen, dette bør også endres på nå som vi bruker den som en lenke. Da bør man i stedet for onClick bruke href. Og da slipper du vel å sette loading state i tillegg.

Reply:
Ikke sant, skal få fikse det på neste PR. Har notert meg ned 😄